solution/planning_solutions.go

```diff
@@ -90,11 +91,15 @@ func FindBestPlanningSolutions(placeClusters [][]matching.Place, topSolutionsCou
 			log.Debug(err)
 			continue
 		}
+		if !plansDeduplication(deduplicatedPlans, candidate) {
```

Comment:
maybe we could rename the function as `isDuplicatePlan`
if isDuplicatePlan() {
    continue;
}

Reply:
Done.